### PR TITLE
Hometask 02.GIT.Local Andrei_Bakaushyn

### PR DESCRIPTION
**Pull request by AndreiBakaushyn**
Good day Sergey. Initially, I misunderstood the tasks, and cloned your repository to my local machine, did all the tasks, and then tried to push them to you on the remote repository in main)) (Erorr 403), that was wery fun ,  then made a fork, and sent a history file).  
With best regards, 
Andrei Bakaushyn

### CODE REVIEW

File: Andrei_Bakaushyn/02.Git.Local/02.GIT.local.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  337  git merge dev
+  338  git merge features
+  339  git cherry-pick hotfix/we_gonna_die
+  340  git checkout dev
+  341  git cherry-pick hotfix/we_gonna_die
+  342  git checkout features
+  343  git cherry-pick hotfix/we_gonna_die
+  344  git checkout master
+  345  history>>02.GIT.Local
+  346  history
```
Comment (pluhin) [inline Andrei_Bakaushyn/02.Git.Local/02.GIT.local.md:69]: Please close code section

**Review by pluhin — CHANGES_REQUESTED** [review]
Missing log of commits for each branch, please add to the end of your report

**Comment by AndreiBakaushyn** [comment]
Fixed report

**Review by pluhin — CHANGES_REQUESTED** [review]
Could you please use git markdown for git log output too

**Comment by AndreiBakaushyn** [comment]
Good day !, 
Covered paragraphs, headings, and corresponding bash command sessions with markdown in my Report

**Pull request closed without merging**